### PR TITLE
Ea 2364 entity retrieval

### PR DESCRIPTION
**Pull request by SrdjanStevanetic**
Please review the changes and merge them.

### CODE REVIEW

File: entity-management-common/src/main/java/eu/europeana/entitymanagement/config/EMSettings.java [hunk 1]
```diff
@@ -26,15 +26,52 @@
  */
 @Configuration
 @PropertySource("classpath:entitymanagement.properties")
-@PropertySource(value = "classpath:myapi.user.properties", ignoreResourceNotFound = true)
+//@PropertySource(value = "classpath:myapi.user.properties", ignoreResourceNotFound = true)
```
Comment (gsergiu) [inline entity-management-common/src/main/java/eu/europeana/entitymanagement/config/EMSettings.java:29]: the sensitive configurations like tokens and datasources must be placed in entitymanagement.user.properties not in entitymanagement.properties

File: entity-management-common/src/main/java/eu/europeana/entitymanagement/config/EMSettings.java [hunk 2]
```diff
@@ -26,15 +26,52 @@
...
+
+	@Value("${authorization.api.name}")
+    private String authorizationApiName;
+    
+    public String getAuthorizationApiName() {
+		return authorizationApiName;
+	}
+
+	@Value("${default.user.token}")
+    private String defaultUserToken;
```
Comment (gsergiu) [inline entity-management-common/src/main/java/eu/europeana/entitymanagement/config/EMSettings.java:66]: there must be no default user token

File: entity-management-common/src/site/site.xml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (gsergiu) [inline entity-management-common/src/site/site.xml:1]: what is the purpose of this file?

File: entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/exceptions/ConceptSchemeProfileValidationException.java [hunk 4]
```diff
@@ -0,0 +1,35 @@
+package eu.europeana.entitymanagement.definitions.exceptions;
```
Comment (gsergiu) [inline entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/exceptions/ConceptSchemeProfileValidationException.java:1]: concept schemes are pot part of entity management, please remove

File: entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/exceptions/UnsupportedAlgorithmTypeException.java [hunk 5]
```diff
@@ -0,0 +1,13 @@
+package eu.europeana.entitymanagement.definitions.exceptions;
+
+public class UnsupportedAlgorithmTypeException extends Exception {
```
Comment (gsergiu) [inline entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/exceptions/UnsupportedAlgorithmTypeException.java:3]: this class is used for entity recommendation which is not part of entity management, please remove this class

File: entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/formats/FormatTypes.java [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+import eu.europeana.entitymanagement.definitions.exceptions.UnsupportedFormatTypeException;
+
+/**
+ * Enumerates the formats
+ * 
+ * @author GrafR
+ *
+ */
+public enum FormatTypes {
```
Comment (gsergiu) [inline entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/formats/FormatTypes.java:11]: we can move this class to api-commons-definitions and depretate the class in Entity api

File: entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/ConceptScheme.java [hunk 7]
```diff
@@ -0,0 +1,50 @@
+package eu.europeana.entitymanagement.definitions.model;
+
+import java.util.Map;
+
+//import eu.europeana.corelib.definitions.edm.entity.ConceptScheme;
+
+public interface ConceptScheme extends Entity{
```
Comment (gsergiu) [inline entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/ConceptScheme.java:7]: ConceptScheme is not part of the EntityManagement,please remove this class

File: entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/RankedEntity.java [hunk 8]
```diff
@@ -0,0 +1,24 @@
+package eu.europeana.entitymanagement.definitions.model;
+
+import java.util.Date;
+
+public interface RankedEntity {
```
Comment (gsergiu) [inline entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/RankedEntity.java:5]: these properties are not part of entity anymore, this is replaced now by the Aggregation interface. Please remove this interface

File: entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/search/SearchProfiles.java [hunk 9]
```diff
@@ -0,0 +1,21 @@
+package eu.europeana.entitymanagement.definitions.model.search;
+
+public enum SearchProfiles {
```
Comment (gsergiu) [inline entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/search/SearchProfiles.java:3]: please remove this enumeration, and merge the code from branch 2326, the EntityProfile enumeration was introduced there

File: entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/search/results/IdBean.java [hunk 10]
```diff
@@ -0,0 +1,25 @@
+package eu.europeana.entitymanagement.definitions.model.search.results;
+
+import java.util.Date;
+
+public interface IdBean {
```
Comment (gsergiu) [inline entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/search/results/IdBean.java:5]: this interface is not needed anymore, please remove

File: entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/vocabulary/LdProfiles.java [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+
+import eu.europeana.entitymanagement.definitions.exceptions.ConceptSchemeProfileValidationException;
+
+/**
+ * This enumeration is intended for Linked Data profiles
+ * 
+ * @author GrafR
+ *
+ */
+public enum LdProfiles implements ProfileKeyword {
```
Comment (gsergiu) [inline entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/vocabulary/LdProfiles.java:11]: please remove this enumeration, we have different specifications, and the Profiles are defined now in EntityProfile enumeration

File: entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/vocabulary/SuggestAlgorithmTypes.java [hunk 12]
```diff
@@ -0,0 +1,18 @@
+package eu.europeana.entitymanagement.definitions.model.vocabulary;
+
+import eu.europeana.entitymanagement.definitions.exceptions.UnsupportedAlgorithmTypeException;
+
+public enum SuggestAlgorithmTypes {
```
Comment (gsergiu) [inline entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/vocabulary/SuggestAlgorithmTypes.java:5]: not used, please remove class

File: entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/view/EntityPreview.java [hunk 13]
```diff
@@ -0,0 +1,57 @@
...
+import java.util.Map;
+
+import eu.europeana.entitymanagement.definitions.model.search.results.IdBean;
+import eu.europeana.entitymanagement.definitions.model.vocabulary.EntityTypes;
+
+/**
+ * @author Sergiu Gordea
+ *
+ */
+public interface EntityPreview extends IdBean {
```
Comment (gsergiu) [inline entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/view/EntityPreview.java:13]: this class is not used in entity management, please remove

File: entity-management-mongo/pom.xml [hunk 14]
```diff
@@ -0,0 +1,118 @@
...
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-data-solr</artifactId>
+        </dependency>
+	
+    
+  </dependencies>
+
+  <build>
+    <pluginManagement><!-- lock down plugins versions to avoid using Maven defaults (may be moved to parent pom) -->
```
Comment (gsergiu) [inline entity-management-mongo/pom.xml:68]: should be moved to the entity-management/pom.xml

File: entity-management-mongo/src/main/resources/fulltext.properties [hunk 15]
```diff
@@ -0,0 +1,41 @@
+# Fulltext properties file
+# Any sensitive data such as username/password settings must be overriden in a file titled fulltext.user.properties
+
+spring.application.name=Fulltext API
```
Comment (gsergiu) [inline entity-management-mongo/src/main/resources/fulltext.properties:4]: this file belongs to the fulltext api, please remove

File: entity-management-mongo/src/site/site.xml [hunk 16]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (gsergiu) [inline entity-management-mongo/src/site/site.xml:1]: what is the purpose of this file?

File: entity-management-utils/src/main/java/eu/europeana/entitymanagement/utils/jsonld/EntityJsonComparator.java [hunk 17]
```diff
@@ -0,0 +1,122 @@
...
+
+/**
+ * This class is specific to annotations must be moved to (europeana) annotation
+ * repository A comparator for JSON-LD maps to ensure the order of certain key
+ * elements like '#', '@', 'a' in JSON-LD output.
+ *
+ * @author Jan Roerden
+ */
+
+public class EntityJsonComparator implements Comparator<Object> {
```
Comment (gsergiu) [inline entity-management-utils/src/main/java/eu/europeana/entitymanagement/utils/jsonld/EntityJsonComparator.java:21]: we will not use Annotation-ld library for serilization but Jackson, please remove this class

File: entity-management-utils/src/main/java/eu/europeana/entitymanagement/utils/jsonld/EuropeanaEntityLd.java [hunk 18]
```diff
@@ -0,0 +1,345 @@
...
+import eu.europeana.entitymanagement.definitions.model.Place;
+import eu.europeana.entitymanagement.definitions.model.Timespan;
+import eu.europeana.entitymanagement.definitions.model.impl.BaseEntity;
+import eu.europeana.entitymanagement.definitions.model.vocabulary.EntityTypes;
+import eu.europeana.entitymanagement.definitions.model.vocabulary.WebEntityConstants;
+import eu.europeana.entitymanagement.definitions.model.vocabulary.WebEntityFields;
+import eu.europeana.entitymanagement.utils.EntityUtils;
+
+
+public class EuropeanaEntityLd extends JsonLd {
```
Comment (gsergiu) [inline entity-management-utils/src/main/java/eu/europeana/entitymanagement/utils/jsonld/EuropeanaEntityLd.java:27]: we will not use Annotation-ld library for serilization but Jackson, please remove this class. See also the JsonLdSerializer class in branch 2326

File: entity-management-utils/src/test/java/org/entity/management/utils/AppTest.java [hunk 19]
```diff
@@ -0,0 +1,38 @@
+package org.entity.management.utils;
+
+import junit.framework.Test;
+import junit.framework.TestCase;
+import junit.framework.TestSuite;
+
+/**
+ * Unit test for simple App.
+ */
+public class AppTest
```
Comment (gsergiu) [inline entity-management-utils/src/test/java/org/entity/management/utils/AppTest.java:10]: please remove this class

File: entity-management-web/src/main/java/eu/europeana/entitymanagement/web/BaseRest.java [hunk 20]
```diff
@@ -0,0 +1,75 @@
...
+    /**
+     * This method selects serialization method according to provided format.
+     * 
+     * @param entity The entity
+     * @param format The format extension
+     * @return entity in jsonLd format
+     * @throws UnsupportedEntityTypeException
+     * @throws HttpException
+     */
+    protected String serialize(EntityRecord entityRecord, FormatTypes format) throws UnsupportedEntityTypeException, HttpException {
```
Comment (gsergiu) [inline entity-management-web/src/main/java/eu/europeana/entitymanagement/web/BaseRest.java:60]: the serialization must use Jackson and not Annotation-LD library. Please use the JsonLdSerializer and XML Serializer

File: entity-management-web/src/main/java/eu/europeana/entitymanagement/web/EMController.java [hunk 21]
```diff
@@ -1,32 +1,109 @@
...
+
+
+	private ResponseEntity<String> createResponse(String type, String namespace, String identifier, FormatTypes outFormat,  String contentType, HttpServletRequest request) throws HttpException{
+	    try {
+	    	
+	    	verifyReadAccess(request);
+	    	
+	    	EntityRecord entity = entityRecordService.retrieveEntityRecordByUri(type, namespace, identifier);
+	    	
+	    	String jsonLd = serialize(entity, outFormat);
```
Comment (gsergiu) [inline entity-management-web/src/main/java/eu/europeana/entitymanagement/web/EMController.java:77]: pleas rename jsonLd property to body

File: entity-management-web/src/main/java/eu/europeana/entitymanagement/web/EMController.java [hunk 22]
```diff
@@ -1,32 +1,109 @@
...
+        
+    	    	Date timestamp = ((RankedEntity)entity.getEntity()).getTimestamp();
```
Comment (gsergiu) [inline entity-management-web/src/main/java/eu/europeana/entitymanagement/web/EMController.java:79]: the timestamp needs to be read from Aggregation object

File: entity-management-web/src/main/java/eu/europeana/entitymanagement/web/model/vocabulary/UserRoles.java [hunk 23]
```diff
@@ -0,0 +1,54 @@
+package eu.europeana.entitymanagement.web.model.vocabulary;
+
+import eu.europeana.api.commons.definitions.vocabulary.Role;
+
+public enum UserRoles implements eu.europeana.api.commons.definitions.vocabulary.Role {
+
+	ANONYMOUS(new String[]{Operations.RETRIEVE}), 
+	USER(new String[]{Operations.RETRIEVE, Operations.CREATE, Operations.DELETE, Operations.UPDATE}), 
+	TESTER(new String[]{Operations.RETRIEVE, Operations.CREATE, Operations.DELETE, Operations.UPDATE}),
```
Comment (gsergiu) [inline entity-management-web/src/main/java/eu/europeana/entitymanagement/web/model/vocabulary/UserRoles.java:9]: doesn't exist anymore, please remove

File: entity-management-web/src/main/java/eu/europeana/entitymanagement/web/model/vocabulary/UserRoles.java [hunk 24]
```diff
@@ -0,0 +1,54 @@
+package eu.europeana.entitymanagement.web.model.vocabulary;
+
+import eu.europeana.api.commons.definitions.vocabulary.Role;
+
+public enum UserRoles implements eu.europeana.api.commons.definitions.vocabulary.Role {
+
+	ANONYMOUS(new String[]{Operations.RETRIEVE}),
```
Comment (gsergiu) [inline entity-management-web/src/main/java/eu/europeana/entitymanagement/web/model/vocabulary/UserRoles.java:7]: doesn't exist anymore, please remove, please remove

File: entity-management-web/src/main/java/eu/europeana/entitymanagement/web/model/vocabulary/UserRoles.java [hunk 25]
```diff
@@ -0,0 +1,54 @@
...
+
+import eu.europeana.api.commons.definitions.vocabulary.Role;
+
+public enum UserRoles implements eu.europeana.api.commons.definitions.vocabulary.Role {
+
+	ANONYMOUS(new String[]{Operations.RETRIEVE}), 
+	USER(new String[]{Operations.RETRIEVE, Operations.CREATE, Operations.DELETE, Operations.UPDATE}), 
+	TESTER(new String[]{Operations.RETRIEVE, Operations.CREATE, Operations.DELETE, Operations.UPDATE}), 
+	ADMIN(new String[]{Operations.RETRIEVE, Operations.CREATE, Operations.DELETE, Operations.UPDATE, Operations.ADMIN_ALL, Operations.ADMIN_UNLOCK, Operations.ADMIN_REINDEX, Operations.USER_CREATE, Operations.USER_DELETE}), 
+	MODERATOR(new String[]{Operations.MODERATION_ALL});
```
Comment (gsergiu) [inline entity-management-web/src/main/java/eu/europeana/entitymanagement/web/model/vocabulary/UserRoles.java:11]: doesn't exist anymore, please remove

File: entity-management-web/src/test/java/eu/europeana/entitymanagement/mongo/InsertAndRetrieveEntityToAndFromMongoDB.java [hunk 26]
```diff
@@ -0,0 +1,73 @@
...
+
+import eu.europeana.entitymanagement.definitions.model.EntityRecord;
+import eu.europeana.entitymanagement.definitions.model.impl.BaseTimespan;
+import eu.europeana.entitymanagement.definitions.model.mongo.impl.EntityRecordImpl;
+import eu.europeana.entitymanagement.mongo.repository.EntityRecordRepository;
+
+/**
+ * JUnit tests to insert and retrieve the entities from the MongoDB
+ */
+public class InsertAndRetrieveEntityToAndFromMongoDB {
```
Comment (gsergiu) [inline entity-management-web/src/test/java/eu/europeana/entitymanagement/mongo/InsertAndRetrieveEntityToAndFromMongoDB.java:22]: please rename to EntityRecrodRepositoryTest, the unit test must be named after tested class/interface

**Review by gsergiu — CHANGES_REQUESTED** [review]
please implement the required changes

File: entity-management-common/src/main/java/eu/europeana/entitymanagement/config/EMSettings.java (hunk 1 above)
Comment (SrdjanStevanetic) [reply entity-management-common/src/main/java/eu/europeana/entitymanagement/config/EMSettings.java:29]: I have added the entitymanagement.user.properties file and distributed the sensitive and non-sensitive properties among them.

File: entity-management-common/src/main/java/eu/europeana/entitymanagement/config/EMSettings.java (hunk 2 above)
Comment (SrdjanStevanetic) [reply entity-management-common/src/main/java/eu/europeana/entitymanagement/config/EMSettings.java:66]: Removed.

File: entity-management-common/src/site/site.xml (hunk 3 above)
Comment (SrdjanStevanetic) [reply entity-management-common/src/site/site.xml:1]: It is a site descriptor, automatically created from maven, for example to include some additional information like logos on top of your site. In our case we do not need it so I removed it.

File: entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/exceptions/ConceptSchemeProfileValidationException.java (hunk 4 above)
Comment (SrdjanStevanetic) [reply entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/exceptions/ConceptSchemeProfileValidationException.java:1]: I miseed to remove it, now it is removed.

File: entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/exceptions/UnsupportedAlgorithmTypeException.java (hunk 5 above)
Comment (SrdjanStevanetic) [reply entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/exceptions/UnsupportedAlgorithmTypeException.java:3]: Removed.

File: entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/formats/FormatTypes.java (hunk 6 above)
Comment (SrdjanStevanetic) [reply entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/formats/FormatTypes.java:11]: I have marked that in the TODO part at the beginning of the file. To do that I would need to replace the api-commons-definitions .jar in the global europeana repository where it is fetched from, for which I would need to have the link.

File: entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/ConceptScheme.java (hunk 7 above)
Comment (SrdjanStevanetic) [reply entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/ConceptScheme.java:7]: Removed.

File: entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/RankedEntity.java (hunk 8 above)
Comment (SrdjanStevanetic) [reply entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/RankedEntity.java:5]: Removed.

File: entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/search/SearchProfiles.java (hunk 9 above)
Comment (SrdjanStevanetic) [reply entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/search/SearchProfiles.java:3]: Removed. I merged your branch into mine.

File: entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/search/results/IdBean.java (hunk 10 above)
Comment (SrdjanStevanetic) [reply entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/search/results/IdBean.java:5]: Removed.

File: entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/vocabulary/LdProfiles.java (hunk 11 above)
Comment (SrdjanStevanetic) [reply entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/vocabulary/LdProfiles.java:11]: Removed.

File: entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/vocabulary/SuggestAlgorithmTypes.java (hunk 12 above)
Comment (SrdjanStevanetic) [reply entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/vocabulary/SuggestAlgorithmTypes.java:5]: Removed.

File: entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/view/EntityPreview.java (hunk 13 above)
Comment (SrdjanStevanetic) [reply entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/view/EntityPreview.java:13]: Removed.

File: entity-management-mongo/pom.xml (hunk 14 above)
Comment (SrdjanStevanetic) [reply entity-management-mongo/pom.xml:68]: Moved.

File: entity-management-mongo/src/main/resources/fulltext.properties (hunk 15 above)
Comment (SrdjanStevanetic) [reply entity-management-mongo/src/main/resources/fulltext.properties:4]: Removed.

File: entity-management-mongo/src/site/site.xml (hunk 16 above)
Comment (SrdjanStevanetic) [reply entity-management-mongo/src/site/site.xml:1]: The same comment as above. Removed.

File: entity-management-utils/src/main/java/eu/europeana/entitymanagement/utils/jsonld/EntityJsonComparator.java (hunk 17 above)
Comment (SrdjanStevanetic) [reply entity-management-utils/src/main/java/eu/europeana/entitymanagement/utils/jsonld/EntityJsonComparator.java:21]: Removed.

File: entity-management-utils/src/main/java/eu/europeana/entitymanagement/utils/jsonld/EuropeanaEntityLd.java (hunk 18 above)
Comment (SrdjanStevanetic) [reply entity-management-utils/src/main/java/eu/europeana/entitymanagement/utils/jsonld/EuropeanaEntityLd.java:27]: Removed.

File: entity-management-utils/src/test/java/org/entity/management/utils/AppTest.java (hunk 19 above)
Comment (SrdjanStevanetic) [reply entity-management-utils/src/test/java/org/entity/management/utils/AppTest.java:10]: Removed.

File: entity-management-web/src/main/java/eu/europeana/entitymanagement/web/BaseRest.java (hunk 20 above)
Comment (SrdjanStevanetic) [reply entity-management-web/src/main/java/eu/europeana/entitymanagement/web/BaseRest.java:60]: Ok.

File: entity-management-web/src/main/java/eu/europeana/entitymanagement/web/EMController.java (hunk 21 above)
Comment (SrdjanStevanetic) [reply entity-management-web/src/main/java/eu/europeana/entitymanagement/web/EMController.java:77]: Done.

File: entity-management-web/src/main/java/eu/europeana/entitymanagement/web/EMController.java (hunk 22 above)
Comment (SrdjanStevanetic) [reply entity-management-web/src/main/java/eu/europeana/entitymanagement/web/EMController.java:79]: Ok.

File: entity-management-web/src/main/java/eu/europeana/entitymanagement/web/model/vocabulary/UserRoles.java (hunk 23 above)
Comment (SrdjanStevanetic) [reply entity-management-web/src/main/java/eu/europeana/entitymanagement/web/model/vocabulary/UserRoles.java:9]: Removed.

File: entity-management-web/src/main/java/eu/europeana/entitymanagement/web/model/vocabulary/UserRoles.java (hunk 24 above)
Comment (SrdjanStevanetic) [reply entity-management-web/src/main/java/eu/europeana/entitymanagement/web/model/vocabulary/UserRoles.java:7]: Removed.

File: entity-management-web/src/main/java/eu/europeana/entitymanagement/web/model/vocabulary/UserRoles.java (hunk 25 above)
Comment (SrdjanStevanetic) [reply entity-management-web/src/main/java/eu/europeana/entitymanagement/web/model/vocabulary/UserRoles.java:11]: Removed.

File: entity-management-web/src/test/java/eu/europeana/entitymanagement/mongo/InsertAndRetrieveEntityToAndFromMongoDB.java (hunk 26 above)
Comment (SrdjanStevanetic) [reply entity-management-web/src/test/java/eu/europeana/entitymanagement/mongo/InsertAndRetrieveEntityToAndFromMongoDB.java:22]: Renamed.

**Pull request merged**